src/components/AllBuildingsList.tsx

```diff
@@ -9,7 +9,7 @@ import { genericSort } from "../utils/genericSort";
 
 //TODO: Maybe move this type since it's used in SavedHomesCard component as well.
 export type BuildingsListProps = {
-  resultBuildingsUnsorted: IBuilding[],
+  resultBuildingsUnsorted?: IBuilding[],
```

Comment:
I made this small change because `yarn build` would not work due to us not passing all the expected props to <SavedHomesList.tsx.